clusters/build-clusters/01_cluster/ci/origin-release/admin_build-starter_rbac.yaml

```diff
@@ -0,0 +1,43 @@
+---
+kind: ServiceAccount
+apiVersion: v1
+metadata:
+  annotations:
+    openshift.io/description: This SA is used to start builds
```

Comment:
That description and name are very useless. How about `origin-release-images-builder`?

Reply:
Updated.